### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by sungam3r**
изображение

### CODE REVIEW

File: BenchmarkTests/combine-bechmarks.csx [hunk 1]
```diff
@@ -10,20 +10,20 @@ var resultsPath = Path.Combine(resultsDir, resultsFile + ".json");
 
 if (!Directory.Exists(resultsDir))
 {
-	throw new DirectoryNotFoundException($"Directory not found '{resultsDir}'");
+	  throw new DirectoryNotFoundException($"Directory not found '{resultsDir}'");
```
Comment (benmwatson) [inline BenchmarkTests/combine-bechmarks.csx:13]: There are lots of spacing changes. What operation are you doing here, exactly?

**Review by benmwatson — CHANGES_REQUESTED** [review]
I'll review again once some of these questions and issues are resolved.

File: .editorconfig [hunk 2]
```diff
@@ -34,3 +34,11 @@ indent_size = 4
 
 csharp_prefer_simple_using_statement = true:warning
 csharp_style_prefer_switch_expression = true:warning
+
+dotnet_style_qualification_for_field = false:warning
```
[the remark this answers is not in the document]
Comment (sungam3r) [reply .editorconfig]: done

File: .editorconfig [hunk 3]
```diff
@@ -34,3 +34,11 @@ indent_size = 4
...
+dotnet_style_qualification_for_property = false:warning
+dotnet_style_qualification_for_method = false:warning
+dotnet_style_qualification_for_event = false:warning
+csharp_style_namespace_declarations = file_scoped:suggestion
```
[the remark this answers is not in the document]
Comment (sungam3r) [reply .editorconfig]: done

File: BenchmarkTests/combine-bechmarks.csx (hunk 1 above)
Comment (sungam3r) [reply BenchmarkTests/combine-bechmarks.csx]: Change indentation from tabs to spaces.

Comment (benmwatson) [reply BenchmarkTests/combine-bechmarks.csx]: ah....I'm actually surprised I didn't notice that. My IDE is set to spaces. Reasonable, thanks!

File: .editorconfig [hunk 4]
```diff
@@ -34,3 +34,11 @@ indent_size = 4
 
 csharp_prefer_simple_using_statement = true:warning
 csharp_style_prefer_switch_expression = true:warning
+
+dotnet_style_qualification_for_field = true:warning
+dotnet_style_qualification_for_property = false:warning
+dotnet_style_qualification_for_method = false:warning
+dotnet_style_qualification_for_event = false:warning
```
Comment (benmwatson) [inline .editorconfig:41]: Sorry if I wasn't clear earlier. These should all be true.

**Review by benmwatson — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 4 above)
Comment (sungam3r) [reply .editorconfig]: done

**Pull request merged**